modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/custom/form-report/index.scss

```diff
@@ -25,12 +25,31 @@
 	}
 }
 
+.col-md-4 {
+	.recharts-pie {
+		font-size: 10px;
+	}
+}
+
+.col-md-4,
+.col-md-6 {
+	.lfr-de__form-report {
+		.card .card-body .legend-container {
+			margin-top: 0;
+			width: 100%;
+		}
+
+		.recharts-responsive-container {
+			width: 100% !important;
```

Comment:
Can you explain to me why the !important was necessary here?

Reply:
After some investigations, we saw this is a technical limitation and we have to use the `!important `on this case indeed. Thanks for the patience Aline! :bowing_woman: